tests/resources/sample_nested_input_schema.json

```diff
@@ -0,0 +1,130 @@
+{
+  "type": "object",
+  "properties": {
+    "param": {
+      "type": "object",
+      "required": [
+        "input1",
+        "input2",
+        "input3"
+      ],
```

Comment:
It looks like you're missing input0

Reply:
After discussion we decide to ignore those items that are not wrapped by any 'ParamterType', this is for the sake of  backward compatibility. For example if none of the items are wrapped, you will get the same schema as before